docs/sc_FunctionalRequirements.csv

```diff
@@ -8,114 +8,114 @@ d)       ATS Command Error Counter
-d)       Command number",SC application needs to validate the table to ensure gross errors are caught.  cFE Table Services handles the ground interface to tables. 
-SC2002,SC2002,SC shall allocate <PLATFORM_DEFINED> Relative Time-tagged Sequences (RTSs) with each capable of storing <PLATFORM_DEFINED>  bytes of stored command data.,Specifies the limits for table sizing purposes.
-SC2002.1,SC2002.1,SC shall resolve time to 1 second for Relative Time Command Sequences (RTS).,Heritage missions have shown that this is adequate resolution.
-SC2002.2,SC2002.2,SC shall accept variable length packed RTS commands within the <PLATFORM_DEFINED> byte relative time-tagged sequences.,
+d)       Command number",SC application needs to validate the table to ensure gross errors are caught.  cFE Table Services handles the ground interface to tables. 
+SC2002,SC2002,SC shall allocate <PLATFORM_DEFINED> Relative Time-tagged Sequences (RTSs) with each capable of storing <PLATFORM_DEFINED>  bytes of stored command data.,Specifies the limits for table sizing purposes.
+SC2002.1,SC2002.1,SC shall resolve time to a wakeup count for Relative Time Command Sequences (RTS).,To address the bug where RTS commands could be delayed when the 1 Hz wakeup occurs near a second rollover.
+SC2002.2,SC2002.2,SC shall accept variable length packed RTS commands within the <PLATFORM_DEFINED> byte relative time-tagged sequences.,
 SC2002.3,SC2002.3,"Each individual command within the sequence shall consist of:
-a)       Time tag with a one second resolution
-b)       A variable length command, with a maximum length of <PLATFORM_DEFINED> bytes",Each command within an RTS needs to contain a time with a one second resolution.
-SC2003,SC2003,"Upon receipt of a table update indication for an RTS Table, SC shall set the RTS status to DISABLED.","Since the RTS definition table does not include a default state, safer to assume disabled."
-SC2004,SC2004,"SC shall execute commands in the ATS table in ascending order, based upon the time-tag of the commands, regardless of the order in which the commands are stored in the ATS table.","Allows flexibility in ground system tools and load generation (command location within an ATS doesn’t matter; Time matters)."
-SC2005,SC2005,SC shall execute no more than <PLATFORM_DEFINED> commands per second from all currently executing RTS tables and/or ATS tables.,Provides bounds on SC processing.
-SC2005.1,SC2005.1,"SC shall defer execution of pending RTS commands, when the combined execution count, of ATS and RTS, exceeds the command per second limit.",
-SC2005.2,SC2005.2,SC shall allow up to the maximum number of defined RTSs to be active concurrently.,RTSs can run concurrently. The limitation is the number of commands that can be sent in one second.
+a)       A relative wakeup count
+b)       A variable length command, with a maximum length of <PLATFORM_DEFINED> bytes",Each command within an RTS needs to contain a wakeup count.

```

Comment:
You can drop the text after the comma for b), this restriction is lifted by moving to wakeup counts. a) already covered needing one.